locations/spiders/essentiel_antwerp.py

```diff
@@ -0,0 +1,29 @@
+import json
+import re
+
+import scrapy
+
+from locations.items import Feature
+
+
+class EssentielAntwerpSpider(scrapy.Spider):
+    name = "essentiel_antwerp"
+    item_attributes = {"brand": "Essentiel-Antwerp"}
+    start_urls = ["https://www.essentiel-antwerp.com/be_fr/storelocator"]
+
+    def parse(self, response):
+        data = response.xpath(".").get()
+        stores = json.loads("[" + re.findall(r"(stores:\[)(.*?)(])", data)[0][1] + "]")
```

Comment:
Meddling with the regex a bit:

- `findall` gets all the instances, but then you are only using `[0]`, `search` should just give us the first.
- You are matching 3 groups, `stores:[`, the stores, and `]`, then adding `[` and `]` back. We can just include them in the match.

```suggestion
        stores = json.loads(re.search(r"stores:(\[.*\])", data).group(1))
```